homeassistant/components/overkiz/water_heater_entities/atlantic_domestic_hot_water_production_mlb_component.py

```diff
@@ -158,10 +158,25 @@ async def async_set_operation_mode(self, operation_mode: str) -> None:
             )
 
     async def async_turn_away_mode_on(self) -> None:
-        """Turn away mode on."""
+        """Turn away mode on. This requires the start date and the end date to be also set."""
+        now_date = cast(
+            dict,
+            self.executor.select_state(OverkizState.CORE_DATETIME),
+        )
         await self.executor.async_execute_command(
-            OverkizCommand.SET_ABSENCE_MODE, OverkizCommandParam.ON
+            OverkizCommand.SET_ABSENCE_MODE,
+            OverkizCommandParam.PROG,
```

Comment:
What is the reason we set this to PROG instead of ON?

Reply:
The Mobile Application for Cozytouch uses 'prog' but as the PR describes, @sippe2 tests concluded that 'on' and 'prog' are equivalent **for now**. I chose to stick with the mobile application for CozyTouch.